client/src/main/java/com/defold/extender/client/ExtenderClient.java

```diff
@@ -47,6 +53,13 @@ public class ExtenderClient {
     public ExtenderClient(String extenderBaseUrl, File cacheDir) throws IOException {
         this.extenderBaseUrl = extenderBaseUrl;
         this.cache = new ExtenderClientCache(cacheDir);
+        this.httpCookies = new BasicCookieStore();
+        this.buildSleepTimeout = Long.parseLong(System.getProperty("com.defold.extender.client.build-sleep-timeout", "5000"));
+        this.buildResultWaitTimeout = Long.parseLong(System.getProperty("com.defold.extender.client.build-wait-timeout", "240000"));
+    }
+
+    private void print(String s, Object... args) {
+        System.out.println(String.format(s, args));
     }
```

Comment:
Should it print unconditionally?

Reply:
Good point. Changed to java.util.logging